JEMS

```diff
@@ -0,0 +1,349 @@
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
+<info>
+<title>Journal of ETA Maritime Science</title>
+<title-short>JEMS</title-short>
+<id>https://csl.mendeley.com/styles/484150931/JEMS</id>
+<link href="https://csl.mendeley.com/styles/484150931/JEMS" rel="self"/>
```

Comment:
also change to: http://www.zotero.org/styles/journal-of-eta-maritime-science

Reply:
Thank you for your reply. I made the changes.